arch/risc-v/src/esp32c3/hardware/esp32c3_soc.h

```diff
@@ -257,4 +257,12 @@
 
 #define BIT(nr)                     (1UL << (nr))
 
+/* Extract the field from the register and shift it to avoid wrong reading */
+
+#define REG_MASK(_reg, _field) ((_reg & (_field##_M)) >> (_field##_S))
```

Comment:
```suggestion
#define REG_MASK(_reg, _field) (((_reg) & (_field##_M)) >> (_field##_S))
```
Wrap macro argument in parentheses to avoid surprises with operator precedence

Reply:
done